### DISCUSSION TITLE
Tunnel Establishment time and Key exchange packets

### DISCUSSION THREAD

**Discussion by vlrk** [strongswan/strongswan#1128] [Q&A]
Hi ,

I am using strongswan 5.9.5 and able to get a tunnel up and working.

two observations:

1. The tunnel Establishment time always changes every 5 min .
Is this expected?. 

2. I see like continuously isakmp packet with ikev2_init[i] on the one of tunnel end point interface and message like " udp port isakmp unreachable". 
Is this any error behaviour?.

As of now when I run ipsec statusall , it gives output with Security Associations and connection established.  

Not sure still , why this key exchange is happening?.

My understanding of tunnel step is as below , 

step1: Key exchange
step2: Using negotiated key exchange set up a SA.
step3: Now on use the negotiated encryption methods to use for traffic exchange.

Correct me if i am missing anything here.

Thanks in advance

**Comment by Thermi** [comment]
1. Depends on your configuration. Read the logs.
2. Check where these packets come from and if they are sent from your peers. Check your firewall rules. On the Internet port scans and protocol scans are common.
